docs/GAME_FLOW.md

```diff
@@ -1379,23 +1356,24 @@ Following are each of the properties available within an FMV.
   <tr valign="top" align="left">
     <th>Property</th>
     <th>Type</th>
-    <th>Required</th>
     <th colspan="2">Description</th>
   </tr>
   <tr valign="top">
     <td>
       <code>path</code>
     </td>
-    <td>String</td>
-    <td>Yes</td>
+    <td>String<strong>*</strong></td>
     <td colspan="2">The path to the FMV's video file.</td>
   </tr>
 </table>
 
+**\*** Required property.
+
```

Comment:
The user config section seems to be getting hidden in the FMV details before it.

Reply:
Addressed